src/RuleDictionnaryOperatingSystem.php

```diff
@@ -72,4 +124,175 @@ public function getActions()
+            'ranking' => '2',
+            'name' => 'Clean Windows OS Name',
+            'match' => 'AND',
+            'is_active' => '0',
+            'is_recursive' => '1',
+            'uuid' => 'clean_windows_os_name',
+            'condition' => '0',
+            'comment' => Sanitizer::sanitize("/(Microsoft)(?>\(R\)|®)? (Windows) (XP|\d\.\d|\d{1,4}|Vista)(™)? ?(.*)/
+
+            Example :
+            Microsoft Windows XP Professionnel -> #1 : Windows
+            Microsoft Windows 7 Enterprise  -> #1 : Windows
+            Microsoft® Windows Vista Professionnel  -> #1 : Windows
+            Microsoft Windows XP Édition familiale  -> #1 : Windows
+            Microsoft Windows 10 Entreprise  -> #1 : Windows
+            Microsoft Windows 10 Professionnel  -> #1 : Windows
+            Microsoft Windows 11 Professionnel  -> #1 : Windows"),
+            'criteria'  => [
+                [
+                    'criteria' => 'os_name',
+                    'condition' => 6,
+                    'pattern' => Sanitizer::sanitize('/(Microsoft)(?>\(R\)|®)? (Windows) (XP|\d\.\d|\d{1,4}|Vista)(™)? ?(.*)/'),
+                ]
+            ],
+            'action'  => [
+                [
+                    'action_type' => 'append_regex_result',
+                    'field' => 'name',
+                    'value' => '#1',
+                ]
+            ]
+        ];
+
+        $rules[] = [
+            'sub_type' => 'RuleDictionnaryOperatingSystem',
+            'ranking' => '3',
+            'name' => 'Clean Windows Server OS Name',
+            'match' => 'AND',
+            'is_active' => '0',
+            'is_recursive' => '1',
+            'uuid' => 'clean_windows_server_os_name',
+            'condition' => '0',
+            'comment' => Sanitizer::sanitize("/(Microsoft)(?>\(R\)|®)? (?:(Hyper-V|Windows)(?:\(R\))?) ((?:Server|))(?:\(R\)|®)? (\d{4}(?: R2)?)(?:[,\s]++)?([^\s]*)(?: Edition(?: x64)?)?$/
+
+            Example :
+            Microsoft Windows Server 2012 R2 Datacenter -> #1 #2 : Windows Server
+            Microsoft(R) Windows(R) Server 2003, Standard Edition x64 -> #1 #2 : Windows Server
+            Microsoft Hyper-V Server 2012 R2 -> #1 #2 : Hyper-V Server
+            Microsoft® Windows Server® 2008 Standard -> #1 #2 : Windows Server"),
+            'criteria'  => [
+                [
+                    'criteria' => 'os_name',
+                    'condition' => 6,
+                    'pattern' => Sanitizer::sanitize('/(Microsoft)(?>\(R\)|®)? (?:(Hyper-V|Windows)(?:\(R\))?) ((?:Server|))(?:\(R\)|®)? (\d{4}(?: R2)?)(?:[,\s]++)?([^\s]*)(?: Edition(?: x64)?)?$/'),
+                ]
+            ],
+            'action'  => [
+                [
+                    'action_type' => 'append_regex_result',
+                    'field' => 'name',
+                    'value' => '#1 #2',
+                ]
+            ]
+        ];
+
+        foreach ($rules as $rule) {
```

Comment:
Call to sanitizer could be done here, instead of doing it multiple times in `$rules[]` declaration.
It will be easier to maintain.

```suggestion
        foreach ($rules as $rule) {
            $rule = Sanitizer::sanitize($rule);
```